### PR TITLE
Require path for node v8 and testcafe-studio is fixed

### PR DESCRIPTION
**Pull request by Dmitry-Ostashev**
https://github.com/DevExpress/testcafe-studio/issues/2734

### CODE REVIEW

File: src/api/test-controller/execution-context.js [hunk 1]
```diff
@@ -6,13 +6,18 @@ import exportableLib from '../exportable-lib';
...
-    const dummyModule = new Module(filename, module);
+    if (Module.createRequire)
+        return Module.createRequire(filename);
+
+    const dummyModule          = new Module(filename, module);
+    const nearbyModulesPath    = dirname(filename).concat('/node_modules');
 
     dummyModule.filename = filename;
-    dummyModule.paths    = [filename].concat(module.paths);
+    dummyModule.paths    = [filename, nearbyModulesPath].concat(module.paths);
```
Comment (AndreyBelym) [inline src/api/test-controller/execution-context.js]: @AlexKamaev, could you please note why we put the filename in `paths`? Also, I think we should put not only `node_modules` from the current directory, but also include all parent `node_modules`: https://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders

Comment (AlexKamaev) [reply src/api/test-controller/execution-context.js]: I rechecked my code and concluded that it's not required to pass filename. @Dmitry-Ostashev 's changes look good to me

File: src/api/test-controller/execution-context.js [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
 import exportableLib from '../exportable-lib';
 
-const OPTIONS_KEY = Symbol('options');
+const OPTIONS_KEY      = Symbol('options');
+const NODE_MODULES_DIR = 'node_modules';
+
+function getModuleBasePaths (currentPath) {
+    const root      = path.parse(currentPath).root;
+    const parentDir = path.dirname(currentPath);
+    const parentPaths = parentDir === root ? [] : getModuleBasePaths(parentDir);
```
Comment (AndreyBelym) [inline src/api/test-controller/execution-context.js]: I think we can avoid recursion here. You can just split `parentDir` by `path.sep`, traverse all path segments and concat `node_modules` in a simple `for` loop.

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

File: src/api/test-controller/execution-context.js (hunk 2 above)
Comment (Dmitry-Ostashev) [reply src/api/test-controller/execution-context.js]: Splitting the dir path by separator leads to different behaviors on Windows and *nix (paths like '/home/username/work...' or 'C:\Work\...')

**Pull request merged**